### PR TITLE
version: Update container image to 16020

### PR DESCRIPTION
**Pull request by amshinde**
This image contains hyperstart changes to not error out in case
there are failures in scanning scsi devices.

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

**Comment by amshinde** [comment]
Required  for  #997

**Comment by jcvenegas** [comment]
lgtm

File: versions.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ json_glib_version=1.2.2
 check_version=0.10.0
 docker_engine_fedora_version=1.12.1
 docker_engine_ubuntu_version=1.12.1-0~xenial
-clear_vm_image_version=15900
+clear_vm_image_version=16020
 clear_vm_kernel_version=4.9.32-71
```
Comment (gorozco1) [inline versions.txt]: we need to sync out also the kernel there when we updated one or the other, so we need the linux-container kernel version that was released in clear image 16020

**Comment by chavafg** [comment]
qa-passed

**Comment by jcvenegas** [comment]
@gorozco1 can you approve this change now ?

**Comment by jcvenegas** [comment]
@amshinde  can we remove the do-not-merge label ?

**Comment by chavafg** [comment]
qa-passed

**Comment by gorozco1** [comment]
lgtm

**Pull request merged**